src/conf_mode/dns_dynamic.py

```diff
@@ -30,6 +30,9 @@
 config_file = r'/run/ddclient/ddclient.conf'
 systemd_override = r'/run/systemd/system/ddclient.service.d/override.conf'
 
+# Dynamic interfaces that might not exist when the configuration is loaded
+dynamic_interfaces = ('peth', 'pppoe', 'sstpc')
```

Comment:
I’m can’t get what is wrong with “peth” interfaces? As it configured statically during the boot process.
Do VLANs have the same issue?
Is it something wrong with priorities?

Reply:
I don't know what's wrong with `peth` either and don't have a way to validate myself. I am going by the forum feedback https://forum.vyos.io/t/issues-loading-config-on-re-boot-when-it-commits-and-runs-fine-beforehand/13000/1

Reply:
I suggest do not exclude (not dynamic ) interfaces under we don’t understand the root cause. It reasonable for PPPoE but not for “peth”, vlans, etc. 
And this way you just “hide” the problem.

Reply:
Agreed 💯